lang/src/org/partiql/lang/syntax/SqlParser.kt

```diff
@@ -2857,7 +2858,10 @@ class SqlParser(private val ion: IonSystem) : Parser {
             else -> node.type.isTopLevelType
         }
         if (topLevelTokenSeen && isTopLevelType) {
-            node.throwTopLevelParserError()
+            when (node.type) {
+                DML_LIST -> node.children.first().throwTopLevelParserError()
+                else -> node.throwTopLevelParserError()
+            }
```

Comment:
`ParseType` of `DML_LIST` has no associated token, so previously if there were multiple dml ops, the error would output a "null" token. This changes to use the first dml op in the list